python/src/ngraph/ops.py

```diff
@@ -2588,3 +2588,83 @@ def reverse_sequence(input, seq_lengths, batch_axis, seq_axis, name=None):
+    # type: (Node, NodeInput, str, bool) -> Node
+    """Return a node which produces the Bucketize operation.
+
+
+    :param data:              Input data to bucketize
+    :param buckets:           1-D of sorted unique boundaries for buckets
+    :param output_type:       Output tensor type, "i64" or "i32", defaults to i64
+    :param with_right_bound:  indicates whether bucket includes the right or left
+                              edge of interval. default true = includes right edge
+    :return: Bucketize node
+    """
+    return _get_node_factory().create('Bucketize', [data, as_node(buckets)],
+        {'output_type': output_type, 'with_right_bound': with_right_bound})
+
+
+@nameable_op
+def range(start, stop, step):
+    # type: (Node, NodeInput, NodeInput) -> Node
+    """Return a node which produces the Range operation.
+
+
+    :param start:  The start value of the generated range
+    :param stop:   The stop value of the generated range
+    :param step:   The step value for the generated range
+    :return: Range node
+    """
+    return _get_node_factory().create('Range', [start, as_node(stop), as_node(step)])
+
+
+@nameable_op
+def region_yolo(input, # type: Node
+                num_coords, # type: int
+                num_classes, # type: int
+                num_regions, # type: int
+                do_softmax,  # type: bool
+                mask, # type: List[int]
+                axis, # type: int
+                end_axis, # type: int
+                anchors = [] # type: List[float]
+                ): # type: Node
+    """Return a node which produces the RegionYolo operation.
+
+
+    :param input:        Input data
+    :param num_coords:   Number of coordinates for each region
+    :param num_classes:  Number of classes for each region
+    :param num_regions:  Number of regions
+    :param do_softmax:   Compute softmax
+    :param mask:         Mask
+    :param axis:         Axis to begin softmax on
+    :param end_axis:     Axis to end softmax on
+    :param anchors:      A flattened list of pairs `[width, height]` that describes prior box sizes
+    :return: RegionYolo node
+    """
+    return _get_node_factory().create('RegionYolo', [input], {
+                                      "coords": num_coords,
+                                      "classes": num_classes,
+                                      "num": num_regions,
+                                      "do_softmax": do_softmax,
+                                      "mask": mask,
+                                      "axis": axis,
+                                      "end_axis": end_axis,
+                                      "anchors": anchors
+                                    })
+
+
+@nameable_op
+def reorg_yolo(input, strides): # type: (Node, NodeInput, NodeInput) -> Node
+    """Return a node which produces the ReorgYolo operation.
+
+
+    :param input:  Input data
+    :param step:   The step value for the generated range
```

Comment:
`step` is not ReorgYolo attribute, add :param `stride` instead
```suggestion
    :param stride:   The distance between cut throws in output blobs.
```